prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics.yaml

```diff
@@ -1253,6 +1253,25 @@ periodics:
       - 'GCP_CLUSTER=multi-repo-8-standard-regular-gitlab'
 
 ### miscellaneous jobs ###
+- <<: *config-sync-ci-job
+  name: multi-repo-kind
+  annotations:
+    testgrid-dashboards: googleoss-kpt-config-sync-misc
+    testgrid-tab-name: multi-repo-kind
+  spec:
+    <<: *config-sync-ci-job-spec
+    containers:
+    - <<: *config-sync-ci-container
+      args:
+      - make
+      - test-e2e-kind-multi-repo
+      resources:
+        requests:
+          memory: "60Gi"
+          cpu: "30000m"
+    nodeSelector:
+      cloud.google.com/gke-nodepool: large-job-pool-period
```

Comment:
`large-job-pool-periodic`

Reply:
oops, thanks for capturing the typo. DONE